Angular/src/app/edit-role-modal/edit-role-modal.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { EditRoleModalComponent } from './edit-role-modal.component';
+
+describe('EditRoleModalComponent', () => {
+  let component: EditRoleModalComponent;
+  let fixture: ComponentFixture<EditRoleModalComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
```

Comment:
Modules/Components/Services will need to be added to get the 'should create' test to run